src/ProjectTemplates/Web.ProjectTemplates/RazorComponentsWeb-CSharp.csproj.in

```diff
@@ -9,7 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.Server" Version="${MicrosoftAspNetCoreComponentsServerPackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="${MicrosoftAspNetCoreMvcNewtonsoftJsonPackageVersion}" />
```

Comment:
Does MVC now support System.Text.Json well enough for us to drop the `Microsoft.AspNetCore.Mvc.NewtonsoftJson` reference too? It's only used for JSON-formatting the `SampleDataController` response.

Reply:
I know that's a bit off-topic for this PR, so please feel free to disregard if you don't want to expand the scope of this change.

Reply:
There's one PR that's pending to allow cleaning this up - https://github.com/aspnet/AspNetCore/pull/8874.

Reply:
I just realised that in this template, we don't even JSON-format anything, because it's all server-side. So we never needed to configure a JSON formatter anyway. I'll remove this in a separate PR.